packages/proto-rpc/src/RpcCommunicator.ts

```diff
@@ -188,7 +188,7 @@ export class RpcCommunicator extends EventEmitter<RpcCommunicatorEvents> {
     private onOutgoingMessage(rpcMessage: RpcMessage, deferredPromises?: ResultParts, callContext?: ProtoCallContext): void {
         if (this.stopped) {
             if (deferredPromises) {
-                const ongoingRequest = new OngoingRequest(deferredPromises, 1000)
+                const ongoingRequest = new OngoingRequest(deferredPromises, 1000, () => {})
```

Comment:
Why we timeout these requests after a second but don't add the `onTimeout` handler?